test/image/mocks/ternary-mathjax.json

```diff
@@ -39,7 +39,15 @@
           "text": "$C^2$"
         },
         "showline": true,
-        "showgrid": true
+        "showgrid": true,
+        "labelalias": {
+          "0": "$0^2$",
+          "20": "$20^2$",
+          "40": "$40^2$",
+          "60": "$60^2$",
+          "80": "$80^2$",
+          "100": "$100^2$"
```

Comment:
Wondering if one could possibly add `$` in prefix and suffix to render every tick by mathjax?

Reply:
Not sure why you would do that, but maybe it would work? Not particularly related to this PR though...

Reply:
To have dynamic mathjax labels when you zoom in & out. 
Illustrated in [c1d127e](https://github.com/plotly/plotly.js/pull/6481/commits/c1d127e98f6b0411fd7c5c884861ef4ea87c8266).